test/conftest.py

```diff
@@ -12,26 +12,26 @@
 elements = [("Lagrange", ufl.interval, 1),
             ("Lagrange", ufl.triangle, 1),
             ("Lagrange", ufl.tetrahedron, 1),
-            ("Lagrange", ufl.quadrilateral, 1),
-            ("Lagrange", ufl.hexahedron, 1),
+            # ("Lagrange", ufl.quadrilateral, 1),
```

Comment:
Why are all these below commented?

Reply:
Quadrature and spectral elements are still not in basix yet, but I've turned the rest back on